### ISSUE TITLE
An issue about the obs

### ISSUE THREAD

**Issue by Zhangwenyao1** [eric-ai-lab/VLMbench#13]
Thanks for your great work!
Are there any segmentation masks or bbox in the dataset?
And if you will consider add these annotations into the dataset or are there interfaces to extract these annotations in this work?

**Comment by KzZheng** [comment]
Yes, there are segmentations in the dataset, and you can find all observation keys at [line 11](https://github.com/eric-ai-lab/VLMbench/blob/dfc3e531bddd933f97d2cdc16f7194d46a608edf/examples/multi_task.py#L11) of examples/multi_task.py
